### DISCUSSION TITLE
ATTinyCore has *discussions* now!

### DISCUSSION THREAD

**Discussion by SpenceKonde** [SpenceKonde/ATTinyCore#508] [General]
This seemed like a good idea. I hope I can get people to actually *use* it! (got any ideas for driving awareness?)

**Comment by bohumir-brocko** [comment]
I'm working on one project that will have an open source for attiny841. If it can be put out, I will recommend your project. If it were also interestingly interesting, I would like to support your work.

**Comment by SpenceKonde** [comment]
Happy to hear that, and best of luck :-) 

If by support you mean the kind involving currency, I am working on getting patreon set up... but there are not enough hours in a day or days in a week.

**Reply by bohumir-brocko** [reply]
Ideally some BTC. So SAT. :)
